cli/dts/lib.deno.ns.d.ts

```diff
@@ -86,94 +112,226 @@ declare interface PerformanceMeasureOptions {
+     * too many redirects were encountered.
+     *
+     * @category Errors */
     export class Http extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying IO resource is not available because it is
+     * being awaited on in another block of code.
+     *
+     * @category Errors */
     export class Busy extends Error {}
-    /** @category Errors */
+    /**
+     * Raised when the underlying Deno API is asked to perform a function that
+     * is not currently supported.
+     *
+     * @category Errors */
     export class NotSupported extends Error {}
   }
 
-  /** The current process id of the runtime.
+  /** The current process ID of this instance of the Deno CLI.
+   *
+   * ```ts
+   * console.log(Deno.pid);
+   * ```
    *
    * @category Runtime Environment
    */
   export const pid: number;
 
   /**
-   * The pid of the current process's parent.
+   * The process ID of parent process of this instance of the Deno CLI.
+   *
+   * ```ts
+   * console.log(Deno.ppid);
+   * ```
    *
    * @category Runtime Environment
    */
   export const ppid: number;
 
   /** @category Runtime Environment */
   export interface MemoryUsage {
+    /** The number of bytes of the current Deno's process resident set size,
+     * which is the amount of memory occupied in main memory (RAM). */
     rss: number;
+    /** The total size of the heap for V8, in bytes. */
     heapTotal: number;
+    /** The amount of the heap used for V8, in bytes. */
     heapUsed: number;
+    /** Memory, in bytes, associated with JavaScript objects outside of the
+     * JavaScript isolate. */
     external: number;
   }
 
   /**
-   * Returns an object describing the memory usage of the Deno process measured
-   * in bytes.
+   * Returns an object describing the memory usage of the Deno process and the
+   * V8 subsystem measured in bytes.
    *
    * @category Runtime Environment
    */
   export function memoryUsage(): MemoryUsage;
 
   /** Reflects the `NO_COLOR` environment variable at program start.
+   *
+   * When the value is `true`, the Deno CLI will attempt to not send color codes
+   * to `stderr` or `stdout` and other command line programs should also attempt
+   * to respect this value.
    *
    * See: https://no-color.org/
    *
    * @category Runtime Environment
    */
   export const noColor: boolean;
 
-  /** @category Permissions */
-  export type PermissionOptions = "inherit" | "none" | PermissionOptionsObject;
+  /**
+   * Options which define the permissions within a test context.
+   *
+   * `"inherit"` would ensure that all permissions of the parent process will be
+   * applied to the test context. `"none"` would ensure the test context has no
+   * permissions. A `TestPermissionOptionsObject` would provide a more specific
+   * set of permissions to the test context.
+   *
+   * @category Testing */
+  export type TestPermissionOptions =
```

Comment:
Just FYI, this interface is also used in `BenchDefintion` and `WorkerOptions`

Reply:
Thanks, I originally could only find usage in testing, but see the unstable worker options, forgot about those.